src/cheri/introduction.adoc

```diff
@@ -30,115 +30,123 @@ be brute forced.
 
-==== Extensions to the unprivileged specification
+Compatibility with RVA, RVB and RVM profiles will be listed in separate {cheri_base_ext_name} based profiles ({cheri_base_ext_name}A, {cheri_base_ext_name}B, {cheri_base_ext_name}M).
+These will list the compatible extensions.
 
-<<rv32y,{cheri_base_ext_name}>>:: Unprivileged specification for {cheri_base_ext_name} to support CHERI.
-<<section_rvy_sentry_insn_ext>>:: Unprivileged extension for capability ambient sealing.
-<<section_cheri_hybrid_ext,{cheri_default_ext_name}>>:: Unprivileged extension to support switching <<cheri_execution_mode>>.
+NOTE: Most existing extensions are compatible with {cheri_base_ext_name} with no modifications.
+ Incompatible extensions will be listed in the profile documents.
 
-The following extension is useful independently of CHERI:
+IMPORTANT: Zicfiss is currently incompatible. An {cheri_base_ext_name} version has not yet been developed.
 
-<<abhlrsc_ext,{lr_sc_bh_ext_name}>>:: Addition of LR.B, LR.H, SC.B, SC.H for more accurate atomic locking as the memory ranges are restricted by using bounds, therefore precise locking is needed.
+This section lists new extensions available to {cheri_base_ext_name}, and also extensions where the behavior is modified for {cheri_base_ext_name}.
 
-==== Privileged specifications
-
-The following privileged specifications are updated for {cheri_base_ext_name}:
-
-<<section_priv_cheri,{cheri_priv_m_ext}>>:: Machine-Level Privileged specification to support CHERI (implies, and required by, Machine-Level ISA).
-<<section_priv_cheri,{cheri_priv_s_ext}>>:: Supervisor-Level Privileged specification to support CHERI (implies, and required by, Supervisor-Level ISA).
-<<section_priv_cheri_vmem,{cheri_priv_vmem_ext}>>:: Privileged specification for integrating CHERI with virtual memory (implies, and required by, by Sv39).
-<<section_priv_cheri,{cheri_priv_h_ext}>>:: Privileged specification to integrate CHERI with the Hypervisor extensions (implies, and required by, H).
-
-The following privileged extensions are added for {cheri_base_ext_name}:
-
-ifdef::support_varxlen[]
-<<section_cheri_dyn_xlen,{cheri_priv_m_dyn_xlen_ext}>>:: Privileged extension to allow dynamic XLEN and endianness changes.
-endif::support_varxlen[]
-<<section_cheri_priv_crg_ext,{cheri_priv_crg_ext}>>^1^:: CHERI extension for capability revocation on RISC-V harts supporting page-based virtual-memory (implies {cheri_priv_vmem_ext}).
+{cheri_base_ext_name} is defined as the base ISA that all CHERI RISC-V implementations must support.
+{cheri_default_ext_name} and {cheri_priv_crg_ext} are examples of optional extensions in addition to
+{cheri_base_ext_name}.
 
-^1^ {cheri_priv_crg_load_tag_ext} is available for improved software revocation performance if {cheri_priv_crg_ext} is implemented.
+In some cases adding an existing extension, such as C or Zca, causes additional instructions to be added.
```

Comment:
```suggestion
In some cases adding an existing extension, such as F or V, causes additional instructions to be added.
```
Maybe better to use different extensions for both cases, C is special in that both cases apply?